Project.toml

```diff
@@ -6,5 +6,6 @@ version = "0.1.0"
 [deps]
 CMake = "631607c0-34d2-5d66-819e-eb0f9aa2061a"
 CxxWrap = "1f15a43c-97ca-5a2a-ae31-89f07a497df4"
+Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
```

Comment:
I a not sure why that was needed?

Reply:
Admittedly, I am not sure either (my commit for that line contains a not-too-confident "[...] seemed to fix it, though I'm not quite sure how"): at one point, when trying to get the build moving forward, I got an error along the lines of cannot import Libdl; this seemed to resolve that error - admittedly, it may have been something spurious.